lib/Controller/OAuthApiController.php

```diff
@@ -199,7 +199,7 @@ public function generateToken($grant_type, $code = null,
 			[
 				'access_token' => $accessToken->getToken(),
 				'token_type' => 'Bearer',
-				'expires_in' => 3600,
+				'expires_in' => $accessToken::EXPIRATION_TIME,
```

Comment:
AccessToken::EXPIRATION_TIME would be better

Reply:
You're absolutely right; adjusted.